src/lib/configs/initConfig.ts

```diff
@@ -32,6 +32,32 @@ import { replaceHomeDir } from "../helpers/replaceHomeDir.js";
 import type { ValidationResult } from "../helpers/validations.js";
 import type { Mutable } from "../typeHelpers.js";
 
+type AjvErrors =
+  | Ajv.ErrorObject<string, Record<string, unknown>, unknown>[]
+  | null
+  | undefined;
+
+const validationErrorToString = (errors: AjvErrors) => {
+  if (errors === null || errors === undefined) {
+    return "";
+  }
+
+  return (
+    "Errors:\n\n" +
```

Comment:
I suggest to use os.EOL instead hardcoding line separator

Reply:
I started renaming yesterday but then I really wanted to know how are we doing on windows front in general at this moment so I installed a windows virtual machine to check this out. Even though there are still problems on windows which require additional development - new line characters does not seem to cause any problems for terminal messages

for files though, I wanna keep it consistent so the new line char is the same across all the systems so git doesn't freak out

So I think the best course of action would be to continue simply using `\n`